src/EnergyPlus/DaylightingManager.cc

```diff
@@ -10394,10 +10414,13 @@ void MapShadeDeploymentOrderToLoopNumber(EnergyPlusData &state, int const enclNu
                 for (auto listOfExtWin : thisDaylightCtrl.ShadeDeployOrderExtWins) {
                     for (auto IWinShdOrd : listOfExtWin) {
                         ++count;
-                        if (count > thisEnclDaylight.NumOfDayltgExtWins)
+                        if (count > thisEnclDaylight.NumOfDayltgExtWins) {
                             ShowWarningError(state,
                                              "MapShadeDeploymentOrderToLoopNumber: too many controlled shaded windows in enclosure " +
                                                  thisSolEnclosureName);
+                            ShowContinueError(state, "Check the list of surfaces in WindowShadingControl: " + thisDaylightCtrl.Name);
```

Comment:
This should be the window shading control name, not `thisDaylightCtrl.Name` Is that known here?

This group of warnings repeats for every surface that's beyond the limit. Not a huge deal, but it would be easier to follow if this just wrote the main warning first, then used the continue error below to list all of the excess surfaces. These first warnings could be moved up to a new if block that checks `if(listOfExtWin.size > thisEnclDaylight.NumOfDayltgExtWins`.